### ISSUE TITLE
Azure icon UTF-8 RTL issue

### ISSUE THREAD

**Issue by itay-grudev** [romkatv/powerlevel10k#2691]
When the Azure icon is displayed the following characters get strangely misaligned, which in the example below move the `4` and the `/` bracket.

![Screenshot from 2024-05-27 03-32-26](https://github.com/romkatv/powerlevel10k/assets/2123767/42099b3f-d9b5-435b-98b3-59fe62f471ed)

This can be reproduced by:

```bash
echo -e "\uFD03" "4"
```

I believe the solution would be to put a `\u200E` UTF-8 Right-to-Left Mark after the Azure symbol. Example:

```bash
echo -e "\uFD03\u200E" "4"
```

**Comment by romkatv** [comment]
If your terminal does that, you need to do the following:

1. Ensure your terminal is configured to use a relatively recent font. As a test, run `print '\UF0805'`. It should print the same icon as `print '\uFD03'`. If it does not, install [the recommended font](https://github.com/romkatv/powerlevel10k/blob/master/font.md).
2. Open `~/.p10k.zsh`, find `POWERLEVEL9K_MODE` and set its value to `nerdfont-v3`.

Note that you can also override individual icons like this:

```zsh
POWERLEVEL9K_AZURE_ICON='\UF0805'
```

Run `get_icon_names` to see the list of all icon labels and their values.

**Comment by itay-grudev** [comment]
@romkatv I am using the recommended font. I think the problem is simply that `\uFD03` (ﴃ) is an Arabic RTL symbol so some symbols after it are also rendered RTL and reversed.

If you don't think the [PR](https://github.com/romkatv/powerlevel10k/pull/2693) is useful though I'll close it.

**Comment by romkatv** [comment]
> @romkatv I am using the recommended font.

It's a new enough version, the confirmation step that I described earlier will succeed.

> I think the problem is simply that `\uFD03` (ﴃ) is an Arabic RTL symbol so some symbols after it are also rendered RTL and reversed.

Yes, I know that. This was an old design mistake in Nerd Fonts that has since been fixed. My previous comment tells you what you can do.

> And `p10k configure` set:
>
> ```zsh
> POWERLEVEL9K_MODE=nerdfont-complete
> ```

This is exactly as I expected. If you do what I suggested, it'll solve your issue.

> If you don't think the [PR](https://github.com/romkatv/powerlevel10k/pull/2693) is useful though I'll close it.

It would be useful if you ensured that it does not break anything on any of the most popular 10 terminals. I don't recommend doing it given the high risk/benefit ratio.

tl;dr: Please try what I suggested in my previous comment.

**Comment by itay-grudev** [comment]
It works of course. Thank you for your time.
